packages/select/src/MenuButton/MenuButton.styles.ts

```diff
@@ -0,0 +1,203 @@
+import { css, cx } from '@leafygreen-ui/emotion';
+import { createUniqueClassName, Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import {
+  focusRing,
+  fontWeights,
+  hoverRing,
+  Size,
+  spacing,
+  typeScales,
+} from '@leafygreen-ui/tokens';
+
+export const menuButtonTextClassName = createUniqueClassName('select-menu');
+
+export const menuButtonStyleOverrides = css`
+  // Override button defaults
+  font-weight: ${fontWeights.regular};
+  > *:last-child {
+    grid-template-columns: 1fr 16px;
+    justify-content: flex-start;
+
+    > svg {
+      justify-self: right;
+      width: 16px;
+      height: 16px;
+    }
+  }
+`;
+
+export const menuButtonSizeStyle: Record<Size, string> = {
+  [Size.Default]: css`
+    > *:last-child {
+      padding: 0 12px;
+    }
+  `,
+  [Size.Large]: css`
+    > *:last-child {
+      padding: 0 12px 0 ${spacing[3]}px;
+    }
+  `,
+  [Size.Small]: css`
+    > *:last-child {
+      padding: 0 ${spacing[2]}px 0 10px;
+    }
+  `,
+  [Size.XSmall]: css`
+    text-transform: none;
+    font-size: ${typeScales.body1.fontSize}px;
+    line-height: ${typeScales.body1.lineHeight}px;
+    > *:last-child {
+      padding: 0 ${spacing[1]}px 0 10px;
+    }
+  `,
+};
+
+export const menuButtonModeOverrides: Record<Theme, string> = {
+  [Theme.Light]: css`
+    background-color: ${palette.white};
+    // Override button default color
+    > *:last-child {
+      > svg {
+        color: ${palette.gray.dark2};
+      }
+    }
+  `,
+  [Theme.Dark]: css`
+    border-color: ${palette.gray.base};
+    background-color: ${palette.gray.dark4};
+    color: ${palette.gray.light3};
+
+    // Override button default color
+    > *:last-child {
+      > svg {
+        color: ${palette.gray.light1};
+      }
+    }
+
+    &:hover,
+    &:active,
+    &:focus {
+      background-color: ${palette.gray.dark4};
+      color: ${palette.gray.light3};
+    }
+  `,
+};
+
+// Override default button focus styles
+export const menuButtonFocusStyle: Record<Theme, string> = {
+  [Theme.Light]: css`
+    &:focus-visible {
+      box-shadow: ${focusRing['light'].input};
+      border-color: rgba(255, 255, 255, 0);
```

Comment:
Should this just be `border-color: transparent`

Reply:
No, we have notes on this somewhere; I can't remember, but transparent in safari is weird.